### ISSUE TITLE
5V support

### ISSUE THREAD

**Issue by gmiranda** [gfoot/picoprom#2]
Do you think it would be feasible to support reading EEPROMs? The [ps2pico project](https://github.com/No0ne/ps2x2pico) uses a logic level converter IC to talk to 5V AT or PS/2 devices, perhaps something like that would work for reading?

**Comment by gfoot** [comment]
Yes in theory you need a level shifter as the Pico's pins are not rated for 5V.  However you might get away without one.  For me the idea here was to not require any glue logic, so I didn't want to do that.

**Comment by gmiranda** [comment]
Thanks for the reply! I might try to use a level shifter and if it works I'd like to collaborate documenting it, if you're open to that
